test_runner/src/main/kotlin/ftl/reports/xml/preprocesor/XmlPreprocessor.kt

```diff
@@ -0,0 +1,14 @@
+package ftl.reports.xml.preprocesor
+
+import org.apache.commons.text.StringEscapeUtils
+
+fun fixHtmlCodes(data: String): String {
+    val isoHtmlCodesToReplace = listOf(0x00..0x1F).union(listOf(0x7F..0x9F)).flatten()
```

Comment:
Is there some document we can reference in a comment about where these magic numbers come from?

Reply:
The other thought I had is, are we sure there's no helper already defined in a library that does what we need? Maybe this is a special case so the custom code is justified.

Reply:
I this case we need to get list of control chars and convert them to html codes with avoid white chars. I cant find method doing this. 
About magic numbers: Im added enum with I hope intuitive names. Enum on the top has comment about destination and link to utf-8 table to prove my idea.